cmd/chat_app_server/main.go

```diff
@@ -29,6 +30,8 @@ func main() {
 
 	handler.NewCustomHandler(api, rt)
 
+	go websocket.RunWebsocketServer(rt)
+
```

Comment:
Looking into this

Reply:
Good to go

Reply:
We need to create a separate main function to run the WebSocket server within the cmd folder, specifically in chat-app-websocket/main.go. Additionally, we should establish a different entry point in the Docker service in your docker-compose file. This will involve defining two separate services, one for the chat server and the other for the WebSocket server.

Reply:
Created a separate main function to run the WebSocket server. Will discuss about Dockerfile and docker-compose file during a call.